src/client/components/form-elements/ReviewCollapsiblePanel.jsx

```diff
@@ -66,16 +106,18 @@ class ReviewCollapsiblePanel extends React.Component {
     }
 
     if (sectionsVerified) {
-      verifiedLabel = (
-        <div className="verify-label">
-          <span className="usa-label">&#10004; I have reviewed</span>
-        </div>);
-    }
-
-    if (sectionsVerified) {
       hiddenSection = (<div></div>);
+      buttonGroup = (<div>
+        <div className="medium-6 columns">
```

Comment:
Are you using this `div` to offset the edit button so it's on the far right? You can use Foundation's offset class for that: `medium-offset-6`. Apply that to the edit button's div instead and then you can remove this one.